proposals/lip-0057.md

```diff
@@ -1984,6 +1984,11 @@ def updateProductivity(b.header: BlockHeader) -> None:
 
 ```python
 def updateSnapshotSubstore(height: uint32)-> None:
+    currentRound = getRoundNumberFromHeight(height)
```

Comment:
For some reason that I can't figure out, the name of this function is not shown in proper color in the LIP. If one of the reviewers knows how to fix this, let me know.

Reply:
I couldn't find any issue.